Day-I-Social-Card/social-card/src/index.js

```diff
@@ -2,5 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import HeaderContainer from '/Users/jeffj/React-UI-Components/Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer';
```

Comment:
Use relative location for it so I think this would be 
`import HeaderContainer from './components/HeaderComponents/HeaderContainer';`